rs/nervous_system/integration_tests/src/pocket_ic_helpers.rs

```diff
@@ -1489,31 +1489,6 @@ pub mod sns {
                 })
         }
 
-        pub async fn get_nervous_system_parameters(
```

Comment:
I think we should keep this.  The hope for this file was to have a library you can use with pocket IC to write tests.  I'm not sure I want everyone to be forced into the same structure of tests.  It's nice to have simple building blocks available.  Maybe making this function work with CallCanisters would be a nice idea?  then it would still work with pocketIC and a principalID?